src/frontend/typer/ressurs.ts

```diff
@@ -40,6 +40,12 @@ export const byggTomRessurs = <T>(): Ressurs<T> => {
     };
 };
 
+export const byggHenterRessurs = <T>(): Ressurs<T> => {
```

Comment:
Fint 👏 

Usikker på om `DataViewer` viser noe fornuftig for denne foreløpig da 🤔

Reply:
Burde jeg legge inn `SystemetLaster`?

Reply:
Ja det hadde vært fint 🚀 Ingen god grunn til at den ikke har vært tatt med

Reply:
e44507adbb023e236039cc70666f37b1b9972082